descartes_moveit/src/moveit_state_adapter.cpp

```diff
@@ -109,7 +109,7 @@ bool MoveitStateAdapter::initialize(const std::string& robot_description, const
     if (world_frame_ != robot_state_->getRobotModel()->getModelFrame())
     {
       logWarn("World frame '%s' does not match model root frame '%s', all poses will be transformed to world frame '%s'",
-               world_frame_.c_str(), link_names.front().c_str(),world_frame_.c_str());
+               world_frame_.c_str(), robot_state_->getRobotModel()->getModelFrame().c_str(),world_frame_.c_str());
```

Comment:
Just a suggestion, but if we change log messages, we should also take the opportunity to update to the ROS macros.